packages/aws-cdk-lib/cx-api/FEATURE_FLAGS.md

```diff
@@ -93,71 +93,7 @@ The following json shows the current recommended set of flags, as `cdk init` wou
 
 <!-- BEGIN json -->
 ```json
-{
```

Comment:
Doesn't seem related to the PR? is this intentional?

Reply:
Nope